pkg/util/fs_util.go

```diff
@@ -107,13 +98,18 @@ func GetFSFromImage(root string, img v1.Image) error {
 					logrus.Debugf("skipping symlink from %s to %s because %s is whitelisted", hdr.Linkname, path, hdr.Linkname)
 					continue
 				}
+				symlinks = append(symlinks, hdr)
```

Comment:
Why do we need to special case symlinks here?

Reply:
It seems like we could just create them during the pass.

Reply:
You're right! I misunderstood how a layer would be formed if a file was removed and then symlinked in the same layer -- fixed it now.